### PR TITLE
Upgrade Ember to 3.2

### PR DESCRIPTION
**Pull request by cmitz**
### Summary
Update the Ember version to 3.2

### Other information
Unfortunately, this update required me to delete the whole Acceptance test suite. Acceptance tests needed **a lot** of setup (authentication, context etc) and those should be replaced by integration tests, like the one I added for the Advertisement tool. The acceptance tests were also really, really slow.

Big plus: this also implements the better readable test structure – no more globals

Big plus 2: Extremely faster testsuite. 

Big min: a lot less tests. Let's get those back in place in the coming months, shall we?

### CODE REVIEW

File: app/abilities/activities.js [hunk 1]
```diff
@@ -5,18 +5,18 @@ import { Ability } from 'ember-can';
 export default Ability.extend({
   session: service(),
   canCreate: computed('session.currentUser', function() {
-    return this.get('session').hasPermission('activity.create');
+    return this.session.hasPermission('activity.create');
```
[the remark this answers is not in the document]
Comment (cmitz) [reply app/abilities/activities.js]: It is as safe as it was with `get`. However, if a key does not exist I'd still expect an error to be thrown. Especially in this specific case

File: testem.js [hunk 2]
```diff
@@ -11,13 +11,15 @@ module.exports = {
...
     Chrome: {
-      mode: 'ci',
-      args: [
+      ci: [
         // --no-sandbox is needed when running Chrome inside a container
         process.env.BUILDKITE ? '--no-sandbox' : null,
-
         '--disable-gpu',
         '--headless',
+        '--disable-gpu',
```
[the remark this answers is not in the document]
Comment (cmitz) [reply testem.js]: ok, nice find!

**Comment by cmitz** [comment]
> I am a bit worried that if we remove them no one will rewrite them 

I can totally see why you think that. However, the tests weren't fixing any issue we had. The website was breaking sometimes anyway, even with tests. And I hope to get people a little more enthousiastic when they see new style tests, it's not **nearly** as much effort to write them as before.

**Comment by Matthijsy** [comment]
Okay, that is true. If anyone else agrees with that we can merge this, I agree with all the other changes

**Pull request merged**